sstables/sstable_version.cc

```diff
@@ -14,12 +14,14 @@ namespace sstables {
 
 const sstring sstable_version_constants::TOC_SUFFIX = "TOC.txt";
 const sstring sstable_version_constants::TEMPORARY_TOC_SUFFIX = "TOC.txt.tmp";
+const sstring sstable_version_constants::INDEX_SUFFIX = "Index.db";
+const sstring sstable_version_constants::DATA_SUFFIX = "Data.db";
```

Comment:
Nit: this change can be refactored into a standalone patch

Reply:
Done